openlibrary/templates/work_search.html

```diff
@@ -165,10 +166,12 @@
             $for work in works:
                 $ ocaid = work.ia[0] if work.ia else None
                 $ availability = (work.get('availability') or {}).get('status')
+                $ users_work_read_status = get_users_read_status(work, username) if work and username else None
+                $ decorations = macros.ReadingLogButton(work, read_status=users_work_read_status, url=url, searchPage="searchPage")
```

Comment:
```suggestion
                $ decorations = macros.ReadingLogButton(work, read_status=users_work_read_status, url=url, form_classes="search-page")
```
Update parameter name and use `kebab-case` for HTML classes and ids.